src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -181,7 +181,7 @@ export class VehicleTechnicalRecordComponent implements OnInit, OnDestroy {
     }
 
     return this.hasTestResultAmend
-      ? 'This vehicle does not have enough information to be tested. Call the Contact Centre to complete this record so tests can be recorded against it.'
-      : 'This vehicle does not have enough information to be tested. Please complete this record so tests can be recorded against it.';
+      ? 'This vehicle does not have enough information to be tested. Please complete this record so tests can be recorded against it.'
```

Comment:
Aren't we not supposed to write Please in error messages 😄

Reply:
Didn't think so but this is the copy that was provided in the ticket.